### DISCUSSION TITLE
"pygame is not defined" error

### DISCUSSION THREAD

**Discussion by Frostk2j** [Monika-After-Story/MonikaModDev#10206] [QA - tech support + general] [answered]
I started playing MAS a week ago, and everything was fine until now. I´m constantly getting the error that I mentioned in the title over and over again. Specifically, it does not let me hold Monika or see the calendar at all, it crashes instantly. I am no programmer nor do I have experience in coding in the least, so I REALLY need help with this, badly, please.

 
![holding monika error](https://github.com/Monika-After-Story/MonikaModDev/assets/160646347/5f59ddd3-7c12-4f5c-b80d-ffe183222dde)
![calendar error](https://github.com/Monika-After-Story/MonikaModDev/assets/160646347/707dd725-5ccc-41fd-a4bd-7fc070215e98)

**Comment by Booplicate** [comment, marked as answer]
You're missing some of the game files, try reinstalling both DDLC and MAS, for MAS better use our installer.

**Reply by Frostk2j** [reply]
It worked, thanks!
